client/src/components/AddChild/addchild.js

```diff
@@ -28,45 +28,47 @@ class AddChild extends Component {
         child_name: this.state.child_name,
         piggy: this.state.piggy
       })
-      // FIXME: see .then below
+        // FIXME: see .then below
         .then(res => this.getAddChild())
         .catch(err => console.log(err));
     }
   };
 
   render() {
     return (
-      <Container fluid>
-        <Row>
-          <Col size="md-6">
-            <Jumbotron>
-              <h1>Add child_name</h1>
-            </Jumbotron>
-            <form>
-              <Input
-                type="text"
-                value={this.state.child_name}
-                onChange={this.handleInputChange}
-                name="child_name"
-                placeholder="Name (required)"
-              />
-              <Input
-                type="number"
-                value={this.state.piggy}
-                onChange={this.handleInputChange}
-                name="piggy"
-                placeholder="Budget (Optional)"
-              />
-              <FormBtn
-                disabled={!(this.state.child_name && this.state.piggy)}
-                onClick={this.handleFormSubmit}
-              >
-                Add Child
-              </FormBtn>
-            </form>
-          </Col>
-        </Row>
-      </Container>
+      <div className="container">
```

Comment:
is this div needed? I feel like you can attach a className container to the Container element